server/middleware/cors.go

```diff
@@ -0,0 +1,17 @@
+package middleware
+
+import (
+	"net/http"
+)
+
+func AddDefaultCORSHeaders(h http.Handler) http.Handler {
```

Comment:
is this the code you were referring to when you said "thanks for the code"? lol

Reply:
lol actually no.